Assets/Scripts/Missions/MissionKitchen1.cs

```diff
@@ -70,9 +70,7 @@ public class MissionKitchen1 : AMission
     private void Awake()
     {
         instantiatedMissionInteractables = new List<GameObject>();
-
-        // TODO: this should probably be changed to a generic enemy type at some point
-        instantiatedEnemies = new List<BaseNavAi>();
+        instantiatedEnemies = new List<BasicNurse>();
```

Comment:
Just to note one of the changes I made to this script involved an iteration over instantiated enemies so you may have to fix the type when you pull from master

Reply:
i have this synced with latest master, unless u have a new pr coming up soon! but yea thatll be fine :)